projects/portal/src/app/pages/mis-turnos/mis-turnos-detalles.component.html

```diff
@@ -0,0 +1,20 @@
+<ng-container *ngIf="turno$ | async as turno">
+    <plex-detail>
+        <plex-icon name="turno-bold" type="info"></plex-icon>
+        <div title case="capitalize" title>{{ turno.horaInicio | date:'dd/MM/yyyy' }} | {{ turno.horaInicio |
+            date:'HH:mm' }}</div>
+        <div subtitle>{{ turno.tipoPrestacion.term }}</div>
+        <plex-badge type="info">{{ turno.horaInicio | date:'dd/MM/yyyy' }}</plex-badge>
+        <plex-badge type="{{turno.asistencia==='asististe' ? 'success' : 'warning' }}">{{ turno.asistencia }}
```

Comment:
Creo que es así, dentro de lo mismo que reportó @palita1991 
```suggestion
        <plex-badge type="{{turno.asistencia==='asistio' ? 'success' : 'warning' }}">{{ turno.asistencia }}
```

Reply:
@andrrrl corregido